hacks/install_etcdctl

```diff
@@ -12,28 +12,54 @@ function show_help () {
 }
 
 function install () {
-  local version=$1
+  # renovate: datasource=github-releases depName=etcd-io/etcd
+  etcd_version="v3.5.12"
```

Comment:
Shall we make the default version to be the one used by Gardener internally i.e. 3.4.26 and if someone wants to override they can do so they can pass in the argument. 

Otherwise for any gardener operator he needs to always do an additional step of running with 3.4.26 which defeats the original purpose.

Reply:
That's a good point.
With the current config we will always get a PR for the latest because of renovate, so if changed we will get PR for the latest anyway.
Where should we read the version, so it's not maintained manually?
Or should we leave it and keep it up manually?

Reply:
Currently we don't upgrade etcd version so frequently, so I'll just keep it manual and when we move to 3.5.x this year as part of this https://github.com/gardener/etcd-druid/issues/445 then we will also adopt the ops-toolbelt accordingly. wdyt?

Reply:
> Otherwise for any gardener operator he needs to always do an additional step of running with 3.4.26 which defeats the original purpose.

ok I guess in this case we should hardcode it and not let it be managed by renovate to update to the latest version. 
Alternatively, @ashwani2k, if the patch version does not matter, we could configure renovate to only update the patch version, like we do it for `kubectl` https://github.com/gardener/ops-toolbelt/blob/13186b58b0bcb0dd183c18f1de892311ff60a95b/.github/renovate.json5#L46

Reply:
I've set it to a hardcoded version.

Reply:
No the patch version matters as we had to upgrade to 3.4.26 from 3.4.13 as it fixed some of the issues which would have forced us to migrate it to 3.5.x instead. 
But i'll ask the etcd colleagues to confirm this, if it really matters or just 3.4 is enough. 
cc @shreyas-s-rao @ishan16696 @aaronfern

Reply:
Generally the patch version doesn't matter from an API perspective. But it's generally recommended to use the same etcdctl version as the etcd that it is used to interact with, since they are well-tested together. In the case of gardener currently, we use v3.4.26 for both the etcd and etcdctl, so we can retain that.

Reply:
As there are no new comments and static version is set, I will resolve this conversation.